Yet another recent breakage in dlls/nsiproxy.sys, yet another patch
submitted upstream.

Plus I've adjusted my autobuilders to focus on newer versions of FreeBSD.


commit cd10f9b73cdae715e4a1ef32ade50c65a097ebf1
Author: Gerald Pfeifer <gerald@FreeBSD.org>
Date:   Mon Sep 6 17:54:53 2021 +0000

    nsiproxy: Include sys/param.h from udp.c.
    
    udp.c features specific code to handle versions of FreeBSD past early
    12.0 snapshots. This is guarded by a check on __FreeBSD_version which
    is defined in sys/param.h. Actually including that, like tcp.c already
    does, activates that check and adjusted code and unbreaks the build.

--- UTC
--- dlls/nsiproxy.sys/udp.c
+++ dlls/nsiproxy.sys/udp.c
@@ -22,6 +22,10 @@
 #include <stdarg.h>
 #include <stddef.h>
 
+#ifdef HAVE_SYS_PARAM_H
+#include <sys/param.h>
+#endif
+
 #ifdef HAVE_SYS_TYPES_H
 #include <sys/types.h>
 #endif
